styles/_base.scss

```diff
@@ -91,20 +104,18 @@ form {
 }
 
 #pageContainer {
-  position: relative;
-
-  // temporary until we have a loading modal
-  &:empty::after {
-    content: "Loading...";
-    font-size: 30px;
-  }
+  padding: 25px;
+  position: relative;  
 }
 
 #chatContainer {
   position: fixed;
   top: 100px;
   right: 0;
   width: $chatClosed;
+  z-index: 1; // chat needs to be on top of modals
+  background-color: rgba(255, 255, 255, 0.6);
```

Comment:
This was just to test to ensure that chat was coming above the modal overlay. I could remove it, but I think it's kind of useful to ensure that any code changes don't fudge that stacking order. Once we actually implement the chat, this could go away, since with the real chat it will be easy to see it's stacking is off.